### DISCUSSION TITLE
Long compile times when changing tailwind.config.js

### DISCUSSION THREAD

**Discussion by afridley** [tailwindlabs/tailwindcss#11934] [Help] [answered]
Im getting rebuild times of 53s whenever adding keyframes to the tailwind.config.js.

Im just running 

`tailwindcss -i ./src/styles/index.css -o ./dist/styles/index.css --watch`

Is this to be expected?

**Comment by adamwathan** [comment, marked as answer]
Hey! Can you share a reproduction? Even on our biggest projects it’s like 300ms. My guess is maybe your `content` configuration is matching an insane number of files, is it a very broad glob?

**Reply by afridley** [reply]
This was 100% the problem must of been scanning node_modules it was so open
